sdcm/sct_config.py

```diff
@@ -502,7 +521,8 @@ class SCTConfiguration(dict):
 
         # Nemesis config options
 
-        dict(name="nemesis_class_name", env="SCT_NEMESIS_CLASS_NAME", type=str,
+        dict(name="nemesis_class_name", env="SCT_NEMESIS_CLASS_NAME",
+             type=_str, k8s_multitenancy_supported=True,
```

Comment:
it's not clear to me, `k8s_multitenancy_supported` is used only for validation ? 
if we are cloning the params object already for each cluster, wouldn't it be easy to make the `params.get` pick the correct value for the cluster ?

Reply:
It is not only validation.
It is qualification role.
To be able to support any type of a config option, not changing type of it I decided to add additional field in one single place - definition of a config option.
I find it pretty clear. Explicit and in single place.
Options that do not have this field set as `True` won't be considered by the multitenant logic.
It started being required after the need to add support for the non-list options.

Reply:
took me a while to find where it's being used, I think it's a good direction. but would rather see the usage of it kept in this code.

Reply:
I don'y understand your proposal.